wiki/Tournaments/oTTMT/1/en.md

```diff
@@ -0,0 +1,285 @@
+---
+tags:
+- oTTMT
+---
+# osu! Tag Team Multiplayer Tournament
+
+![oTTMT Logo](img/logo.png)
+
+The **osu! Tag Team Multiplayer Tournament** (***oTTMT***) was a two-on-two tag team based osu! tournament hosted by ![][flag_SE] [Gabi](/users/57057). It was the 1st installment of the osu! Tag Team Multiplayer Tournament.
+
+## Tournament Schedule
+
+- The osu! Tag Team Multiplayer Tournament started 2009-06-12 and ended 2009-09-06
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for osu! Tag Team Multiplayer Tournament winners. It is possible that these items change with every installment of the osu! Tag Team Multiplayer Tournament.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter tag, profile badge |
+
+## Organisation
+
+The osu! Tag Team Multiplayer Tournament was run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Management | ![][flag_SE] [Gabi](/users/57057) |
+| Referees | ![][flag_SE] [Gabi](/users/57057), ![][flag_US] [Daru](/users/32480), ![][flag_DE] [Larto](/users/12328), ![][flag_US] [EEeee](/users/19819) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/12710)
+
+## Participants
+
+| Team | Members |
+| :-- | :-- |
+| **Does it matter?** | Gopice, Livender |
+| **Less QQ more PEWPEW** | Sir Minelli, Tobebuta |
+| **Fagboat** | Hawkzor, Lucidity |
+| **Mlekołaki** | CrazyBanana, skratch |
+| **Huge C\*cks** | Rolled, Foulcoon |
+| **Banchobag** | Lesjuh, HenkieBP |
+| **IITF** | WyndII, RaisTF |
+| **MINMIrox** | DeathxShinigami, Zatoichi |
+| **Mitizuma** | Inozuma, Mitano |
+| **Asskickers!** | SteRRuM, H1ko |
+| **peppyfangirlsamirite** | Rena-chan, Mr\_Tinkles |
+| **two noobs** | himydanny, x2652300 |
+| **Fumoffu** | Dangaard, happy30 |
+| **The Craweszoymators.** | Celestral & Ten |
+| **Lubimy Ponczgi!** | Ryohan, SeymourLyon |
+| **Appelmoes** | Gladi, Remco32 |
+| **Let's rock** | Doomsday93, Hanyuu |
+| **Allstar** | Alace, Moonstar |
+| **Not Ready** | Starrodkirby86, Real1 |
+| **100% Failure** | Jurgan, Alex0686 |
+| **Animalz D:** | Niedzwiedz1124, White wolf |
+| **⁈** | Derekku Chan, strager |
```

Comment:
replace `⁈` with `?!` anywhere it appears in this article, the osu! wiki font doesn't support it